### PR TITLE
add support for isContinuous

### LINKED ISSUE #383: improve support for "Apply/Build/Update" button in Variable
add an option to switch off the callback for ui elements, proposal add `isContinuously` (default to 
`True` to keep the current behaviour.

```python
Variable([
    dict(name="foo", ui="Slider", isContinuously=False),
    dict(name="Update", ui="Button")
], globals())
```

### PR DESCRIPTION
**Pull request by typemytype**
option to disable a direct callback of sliders and other control changes

fixes #383

### CODE REVIEW

**Comment by typemytype** [comment]
test are fixed in different PR...

**Comment by gferreira** [comment]
⚠️ `isContinuous`

File: drawBot/drawBotDrawingTools.py [hunk 1]
```diff
@@ -2369,14 +2369,18 @@ def ImageObject(self, path=None):
...
         """
         Build small UI for variables in a script.
 
         The `workSpace` is usually `globals()`
         as you want to insert the variable in the current workspace.
         It is required that `workSpace` is a `dict` object.
 
+        Setting `isContinuous` to `False` will add a "Update" button at the bottom.
+        for the user to click to get and update. The default is `True` and will
+        set all control updating the script continuously.
```
Comment (justvanrossum) [inline drawBot/drawBotDrawingTools.py:2382]: This needs editing. Perhaps:
"""
The `isContinuous` argument controls whether whether script is run when UI elements change. The default is `True`, which will execute the script immediately and continuously when the user input changes. When set to `False`, there will be an "Update" button added at the bottom of the window. The user will have to click this button to execute the script and see the changes. This is useful when the script is slow, and continuous execution would decrease responsiveness.
"""

Comment (justvanrossum) [reply drawBot/drawBotDrawingTools.py:2382]: Btw. I think I would still prefer to call it `continuous`, to be more consistent with Vanilla.

Comment (typemytype) [reply drawBot/drawBotDrawingTools.py]: changed!

File: drawBot/drawBotDrawingTools.py [hunk 2]
```diff
@@ -2369,14 +2369,22 @@ def ImageObject(self, path=None):
...
-    def Variable(self, variables, workSpace):
+    def Variable(self, variables, workSpace, continuous=True):
         """
         Build small UI for variables in a script.
 
         The `workSpace` is usually `globals()`
         as you want to insert the variable in the current workspace.
         It is required that `workSpace` is a `dict` object.
 
+        The `continuous` argument controls whether whether script is run when UI
```
Comment (justvanrossum) [inline drawBot/drawBotDrawingTools.py:2380]: "whether whether script" -> "whether the script"

File: drawBot/drawBotDrawingTools.py [hunk 3]
```diff
@@ -2369,14 +2369,22 @@ def ImageObject(self, path=None):
...
-    def Variable(self, variables, workSpace):
+    def Variable(self, variables, workSpace, continuous=True):
         """
         Build small UI for variables in a script.
 
         The `workSpace` is usually `globals()`
         as you want to insert the variable in the current workspace.
         It is required that `workSpace` is a `dict` object.
 
+        The `continuous` argument controls whether script is run when UI
```
Comment (justvanrossum) [inline drawBot/drawBotDrawingTools.py:2380]: "whether script is" -> "whether the script is"

**Pull request merged**